contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--1.0.0--1.1.0.sql

```diff
@@ -2,6 +2,8 @@
 \echo Use "ALTER EXTENSION ""babelfishpg_tsql"" UPDATE TO '1.1.0'" to load this file. \quit
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+-- enable DDL from pgendpoint
+SELECT set_config('babelfishpg_tsql.enable_ddl_from_pgendpoint', 'true', false);
```

Comment:
can we set it for whole create/update extension duration to avoid set/reset at multiple places

Reply:
For create extension, We are setting/resetting at babelfishpg_tsql.in . But for for upgrade we've to add it to every script.